src/main/java/com/robosh/ejournal/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,25 @@
+package com.robosh.ejournal.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.NOT_FOUND)
+public class ResourceNotFoundException extends RuntimeException {
```

Comment:
I think it may be enough, and you shouldn't write at controller ExceptionHandler